releases/release-timeline.md

```diff
@@ -0,0 +1,136 @@
+| [3.0.0](/releases/release-3.0-ga.md) | 2019-06-28 |
+| [3.0.0-rc.3](/releases/release-3.0.0-rc.3.md) | 2019-06-21 |
+| [2.1.13](/releases/release-2.1.13.md) | 2019-06-21 |
+| [2.1.12](/releases/release-2.1.12.md) | 2019-06-13 |
+| [2.1.11](/releases/release-2.1.11.md) | 2019-06-03 |
+| [3.0.0-rc.2](/releases/release-3.0.0-rc.2.md) | 2019-05-28 |
+| [2.1.10](/releases/release-2.1.10.md) | 2019-05-22 |
+| [3.0.0-rc.1](/releases/release-3.0.0-rc.1.md) | 2019-05-10 |
+| [2.1.9](/releases/release-2.1.9.md) | 2019-05-06 |
+| [2.1.8](/releases/release-2.1.8.md) | 2019-04-12 |
+| [2.1.7](/releases/release-2.1.7.md) | 2019-03-28 |
+| [3.0.0-beta.1](/releases/release-3.0.0-beta.1.md) | 2019-03-26 |
+| [2.1.6](/releases/release-2.1.6.md) | 2019-03-15 |
+| [2.1.5](/releases/release-2.1.5.md) | 2019-02-28 |
+| [2.1.4](/releases/release-2.1.4.md) | 2019-02-15 |
+| [2.1.3](/releases/release-2.1.3.md) | 2019-01-28 |
+| [3.0.0-beta](/releases/release-3.0-beta.md) | 2019-01-19 |
+| [2.0.11](/releases/release-2.0.11.md) | 2019-01-03 |
+| [2.1.2](/releases/release-2.1.2.md) | 2018-12-22 |
+| [2.0.10](/releases/release-2.0.10.md) | 2018-12-18 |
+| [2.1.1](/releases/release-2.1.1.md) | 2018-12-12 |
+| [2.1.0](/releases/release-2.1-ga.md) | 2018-11-30 |
+| [2.0.9](/releases/release-2.0.9.md) | 2018-11-19 |
+| [2.1.0-rc.5](/releases/release-2.1-rc.5.md) | 2018-11-12 |
+| [2.1.0-rc.4](/releases/release-2.1-rc.4.md) | 2018-10-23 |
+| [2.0.8](/releases/release-2.0.8.md) | 2018-10-16 |
+| [2.1.0-rc.3](/releases/release-2.1-rc.3.md) | 2018-09-29 |
+| [2.1.0-rc.2](/releases/release-2.1-rc.2.md) | 2018-09-14 |
+| [2.0.7](/releases/release-2.0.7.md) | 2018-09-07 |
+| [2.1.0-rc.1](/releases/release-2.1-rc.1.md) | 2018-08-24 |
+| [2.0.6](/releases/release-2.0.6.md) | 2018-08-06 |
+| [2.0.5](/releases/release-2.0.5.md) | 2018-07-06 |
+| [2.1.0-beta](/releases/release-2.1-beta.md) | 2018-06-29 |
+| [2.0.4](/releases/release-2.0.4.md) | 2018-06-15 |
+| [2.0.3](/releases/release-2.0.3.md) | 2018-06-01 |
+| [2.0.2](/releases/release-2.0.2.md) | 2018-05-21 |
+| [2.0.1](/releases/release-2.0.1.md) | 2018-05-16 |
+| [2.0.0](/releases/release-2.0-ga.md) | 2018-04-27 |
+| [2.0.0-rc.5](/releases/release-2.0-rc.5.md) | 2018-04-17 |
+| [2.0.0-rc.4](/releases/release-2.0-rc.4.md) | 2018-03-30 |
+| [2.0.0-rc.3](/releases/release-2.0-rc.3.md) | 2018-03-23 |
+| [2.0.0-rc.1](/releases/release-2.0-rc.1.md) | 2018-03-09 |
+| [1.1.0-beta](/releases/release-1.1-beta.md) | 2018-02-24 |
+| [1.0.8](/releases/release-1.0.8.md) | 2018-02-11 |
+| [1.0.7](/releases/release-1.0.7.md) | 2018-01-22 |
+| [1.1.0-alpha](/releases/release-1.1-alpha.md) | 2018-01-19 |
+| [1.0.6](/releases/release-1.0.6.md) | 2018-01-08 |
+| [1.0.5](/releases/release-1.0.5.md) | 2017-12-26 |
+| [1.0.4](/releases/release-1.0.4.md) | 2017-12-11 |
+| [1.0.3](/releases/release-1.0.3.md) | 2017-11-28 |
+| [1.0.2](/releases/release-1.0.2.md) | 2017-11-13 |
+| [1.0.1](/releases/release-1.0.1.md) | 2017-11-01 |
```

Comment:
These are broken links because these release notes do not have a Chinese version. Can we link to the English notes using the HTTP absolute path?

Reply:
Yes. Will use the English links.